src/config/schema.ts

```diff
@@ -353,6 +353,13 @@ export const configSchema = {
       required: ["mandatory"],
       additionalProperties: false,
     },
+    skipFeeProbe: {
+      type: "object",
+      patternProperties: {
+        "^.*$": { type: "array", items: { type: "string" }, uniqueItems: true },
```

Comment:
length could be fix to 66 (IIRC) characters?

Reply:
BOLT-11 says these are 33-bytes, so if it's hex-encoded then 66 sounds right 👍🏾

Reply:
Done: https://github.com/GaloyMoney/galoy/pull/1505/commits/64efd9298363f055547d51669820da57537081ef